components/SimpleCrossfadeImages.js

```diff
@@ -0,0 +1,77 @@
+// Simple crossfade images
+// credit: http://css3.bradshawenterprises.com/cfimg/
+
+const images = [
+  'static/hero-bg.jpg',
+  'static/hero-bg-2.jpg',
+  'static/hero-bg-3.jpg'
+]
+export default class SimpleCrossfadeImages extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      currentImageIndex: 0
+    }
+    this.tick = this._tick.bind(this)
+  }
+  componentDidMount () {
+    setTimeout(this.tick, this.props.timeout)
+  }
+  _tick () {
+    setTimeout(this.tick, this.props.timeout)
+  }
```

Comment:
Line 10~22 doesn’t seem to be used. You can turn this into stateless functional component. :)